File: problems/leetcode/2469-convert_the_temperature.cpp
class Solution {
public:
    vector<double> convertTemperature(double celsius) {
        double kConv = celsius + 273.15;
        double fConv = celsius * 1.80 + 32.00;

        vector<double> res = {kConv, fConv};
        return res;
    }
};

File: solutions/leetcode/2469-convert_the_temperature.py
class Solution(object):
    def convertTemperature(self, celsius):
        """
        :type celsius: float
        :rtype: List[float]
        """
        kConv = celsius + 273.15
        fConv = celsius * 1.80 + 32.00

        return [kConv, fConv]
        